src/ui/setup/dynamic/devtools.ts

```diff
@@ -223,19 +221,6 @@ export default async function setupDevtools(store: AppStore, replayClient: Repla
   // Add protocol event listeners for things that the Redux store needs to stay in sync with.
   // TODO We should revisit this as part of a larger architectural redesign (#6932).
 
-  setMouseDownEventsCallback(
-    // We seem to get duplicate mousedown events each time, like ["a"], ["a"], ["a", "b"], ["a", "b"], etc.
-    // Debounce the callback so we only dispatch the last set.
-    debounce((events: MouseEvent[]) => {
```

Comment:
TBH I don't know what this was referencing. I don't see duplicate events when I compare this branch to production but I guess we can keep an eye out for this.